src/client/pages/Geo/GeoMapWrapper/geoMapWrapper.scss

```diff
@@ -30,4 +30,10 @@
     flex-direction: column;
     z-index: 1;
   }
+  &.statistics {
+    // Makes the statistics divs be the same length as the button.
+    > div {
+      width: 100%;
+    }
```

Comment:
can this be defined in the inner components or even update the layout of the .statistics element ?

Reply:
Yes, done.